sklearn/model_selection/tests/test_split.py

```diff
@@ -667,6 +667,15 @@ def test_stratified_shuffle_split_overlap_train_test_bug():
     assert_array_equal(np.intersect1d(train, test), [])
 
 
+def test_stratified_shuffle_split_multilabel():
+    # issue 9037
+    for y in [np.array([[0, 1], [1, 0], [1, 0], [0, 1]]),
```

Comment:
I could build it explicitly with a (Multi)LabelBinarizer if it makes the test clearer?